from0k2bp.tex

```diff
@@ -1431,30 +1498,31 @@
 smaller vectors -- in our concrete example, they started with a vector
 of dimension 10, it's now a vector of dimension 2. If the process is to
 end there, the Prover will simply reveal this vector (here it would be
-the components $(xa_1 + x^2 a_3 + x^3 a_5 + x^4 a_7 + x^5 a_9), (xa_2 + x^2 a_4 + x^3 a_6 + x^4 a_8 + x^5 a_{10})$ , as two numbers).
+the components 
+$(xa_1 + x^2 a_3 + x^3 a_5 + x^4 a_7 + x^5 a_9), (xa_2 + x^2 a_4 + x^3 a_6 + x^4 a_8 + x^5 a_{10})$, 
+as two numbers).
 
 But if we started with, say, a vector of dimension 600, we could have
 followed the above procedure as 60x10, and at this point the basis
 vectors $\mathbf{G}'$ may have dimension 60, and we could repeat, chunking 60 = 6 x 10
 again. We would repeat the above algorithm, except replacing $A \rightarrow A' \ ,\ \mathbf{G} \rightarrow \mathbf{G}'$.
 
 Summarizing: in the case with multiple reductions, the pattern of
-interaction would be : P sends diagonal commitments, V sends $x$, (both
-sides calculate next step), P sends new diagonal commitments, V sends $x'$,
-\ldots{} , last step: P sends full commitment openings to a small
-vector.
+interaction would be: $\mathcal{P}$ sends diagonal commitments, $\mathcal{V}$ sends $x$, (both
+sides calculate next step), $\mathcal{P}$ sends new diagonal commitments, $\mathcal{V}$ sends $x'$,
+\ldots{}, last step: $\mathcal{P}$ sends full commitment openings to a small vector.
 
 So has this admittedly complicated algorithm helped us? Let's count: for
 the dimension-10 case, first we had to send an additional 9 - 1
 commitments (subtract off the main diagonal; that's $A$, which we
 transferred at the start), then we revealed 2 scalars at the end. So
-that's actually \textasciitilde{} the same as the 10 scalars we'd have
+that's actually about the same as the~10 scalars we'd have
 to reveal if we did it without shenanigans.
 
 But for the case $600 = 10 \times 10 \times 6$ - we first ``chunk'' in 10s, then
 again in 10s, leaving only 6 components for the final step. That
 requires revealing $2\times 10-1 = 19$ commitments at each of the two reducing
-steps, along with 6 scalars in the final step (and again subtract 1 for
+steps, along with 6~scalars in the final step (and again subtract 1 for
```

Comment:
OK I see, this is a repeating pattern again. Don't put a tilde between a number and the thing it refers to.

Reply:
Oh. Now I'm thinking it must be my ignorance at work - this is some LaTeX formatting feature? Oops, in that case you can ignore the 4 or so comments I made about it.

Reply:
Yeah. I have explained the use of tilde in my previous comments.